Main page styles

Androborn opened the pull request:
**Main page adaptive styles and general minor changes to project**

*Adds adaptive styles for Main page sections:
- hero
- gallery
- more-photo

*Adds adaptive bg images to the main page.

*Adds icon-loades sprite - applied in more-photo section bottom.

*Changes desktop breakpoint for adaptive images in html from 1444px to 1440px.

**!!!*Changes general container dimensions in _base.scss**

**!!!Background is set to <div class="overlay main__overlay"> with background-position properties in _hero.scss - must be set to main page <body> instead with positioning properties removed.**

Androborn committed:
Start adding styles

Androborn committed:
Main page adaptive styles and general minor changes to project

*Adds adaptive styles for Main page sections:
- hero
- gallery
- more-photo

*Adds adaptive bg images to the main page.

*Changes generan container dimentions in _base.scss

*Adds icon-loades sprite - applied in more-photo section bottom.

*Changes desktop brakepoint for adaptive imeges in html from 1444px to 1440px.

!!!Backgroung is set to <div class="overlay main__overlay"> with background-position properties in _hero.scss - must be set to main page <body> instead with positioning properties removed.

codajoker commented on src/sass/base/_base.scss:
```diff
@@ -59,31 +59,25 @@ img {
-    padding-left: 20px;
-    padding-right: 20px;
-  }
+  min-width: 320px;
```
Базовые стили уже изменили

Androborn replied:
Запушил комит с правкой стилей контейнера:
- общие вернул твои
- на свою страницу задал отдельные

codajoker commented on src/sass/layout/main/_hero.scss:
```diff
@@ -0,0 +1,167 @@
+  margin-left: auto;
+  margin-right: auto;
+  background-position: center -93px;
+  background-image: url(../../src/images/mobile/main/page_bg.jpg);
```
пути ../

Androborn replied:
Запушил комит с правкой.

Androborn committed:
Update _more_photo.scss

*Corrects erroneous concatination.

Androborn committed:
Container geometry correction

*Returns project container common properties to _base.scss.

*Adds specific container properties to main pages sections:
- hero
- gallery
- more photo

Androborn committed:
Update _hero.scss

*Changes bg images path from ../../src/images/ to ../images/
